fix: fix small errors (#7)
waw-muzik/rfcs: active-rfcs/0004-api-versioning.md

# Motivation

> We want to improve our API and business logic adding more semantic data, performance,how we use the database, ..., In perder to avoid to break something, semantic versioning will be the best approach for this. For now, we will only use major versions, so that all clients will be using something like v1, v2, v3 ...

Reviewer: ```suggestion
We want to improve our API and business logic adding more semantic data, performance, how we use the database, ..., In order to avoid to break something, semantic versioning will be the best approach for this. For now, we will only use major versions, so all clients will be using something like v1, v2, v3 ...
```

curl -i -H " Accept: application/vnd.wawmuzik.v1+json" -H "Content-Type: application/json" http://app.wawmuzik.com/auth/login
```

the response will come with this type of header:

```bash
Content-Type: application/vnd.wawmuzik.v1+json
```

> In backend, in our routers, we will need to check this header before doing anything in the endpoint, so we will have to make a Factory that manages this behaviour.

Reviewer: ```suggestion
In the backend, in our routers, we will need to check this header before doing anything in the endpoint. This will be managed by a Factory, which must be implemented soon.
```

> For now, requests that don't have this header will be redirected to the first version of our API. This approach will be for now, because in the future, when we will remove this 1 version, requests that don't have this header will be redirected to the lastest version

Reviewer: ```suggestion
For now, requests without this header will be redirected to the API's first version. This approach will be for now, for giving backward compatibility to the v1. When v1 becomes deprecated or gets removed, requests without this header will be redirected to the latest version
```
